cornice/service.py

```diff
@@ -281,9 +277,14 @@ def add_view(self, method, view, **kwargs):
         args = self.get_arguments(kwargs)
 
         # These attributes were used in views in Cornice < 1.0.
-        deprecated_attrs = ('acl', 'factory', 'traverse')
+        deprecated_attrs = ('acl', 'traverse')
```

Comment:
I think `acl` does not have to be here since we raise if it is specified. We could do the same as above with `self.traverse`, since we bump a major

Reply:
Will `raise ConfigurationError()` for `traverse` as well and upgrade rSTs.